.circleci/api-comment.js

```diff
@@ -53,7 +53,13 @@ async function run() {
 
   if (pr != null) {
     let commentId = await findDifferComment(pr);
-    let diffs = fs.readFileSync('/tmp/dist/ts-diff.txt');
+    let diffs;
```

Comment:
this is just to make it easier when testing verdaccio on a branch, you don't need to worry about there being a ts-diff file